### PR TITLE
Release Couchbase Server 7.0.1 (EE and CE)

### PR DESCRIPTION
**Pull request by mhocouchbase**

### CODE REVIEW

File: library/couchbase [hunk 1]
```diff
@@ -1,11 +1,19 @@
...
-Tags: latest, enterprise, 7.0.0, enterprise-7.0.0
+Tags: latest, enterprise, 7.0.1, enterprise-7.0.1
+GitCommit: f412542829939319b5e722fa384c60b68e765bce
+Directory: enterprise/couchbase-server/7.0.1
+
+Tags: community, community-7.0.1
+GitCommit: f412542829939319b5e722fa384c60b68e765bce
+Directory: community/couchbase-server/7.0.1
+
+Tags: 7.0.0, enterprise-7.0.0
```
Comment (tianon) [inline library/couchbase:12]: Any particular reason to keep this version listed here?  Removing tags here will remove them from the "Supported" section on the Hub readme (and will prevent us from spending cycles rebuilding them on the official build servers), but the tags will still be available to users who want them.  (See https://github.com/docker-library/official-images#library-definition-files for more detail on this.)

File: library/couchbase [hunk 2]
```diff
@@ -1,11 +1,19 @@
...
 GitCommit: a4a4b016732b4ea3a42ea115f481c66ba69b980d
 Directory: enterprise/couchbase-server/7.0.0
 
-Tags: community, community-7.0.0
+Tags: community-7.0.0
```
Comment (yosifkit) [inline library/couchbase:16]: Any particular reason to keep this version listed here?  Removing it here will remove it from the "Supported" section on the Hub readme (and will prevent us from spending cycles rebuilding it on the official build server), but the tag will still be available to users who want it.  (See https://github.com/docker-library/official-images#library-definition-files for more detail on this.)

**Comment by tianon** [comment]
Friendly ping?

(cc @ceejatec @cb-robot)

**Comment by mhocouchbase** [comment]
@tianon, I have pushed an update to remove 7.0.0 tags.

**Comment by tianon** [comment]
Thanks, merging!

(CI failures are the known 6.x issues.)

**Pull request merged**